src/puppetlabs/ezbake/core.clj

```diff
@@ -847,22 +847,25 @@ Additional uberjar dependencies:
 (defmethod action "build"
   [_ lein-project build-target]
   (action "stage" lein-project build-target)
+  (exec/exec "bundle" "install" :dir staging-dir)
```

Comment:
Do we also need to specify a `--path` flag here as to install to a self contained gem path?

Reply:
Is that not what `:dir staging-dir` is for?

Reply:
it might not be

Reply:
I am bad at Clojure

Reply:
@mwaggett `:dir staging-dir` doesn't have anything to do with the `--path` like you have in other bundle installs, the `:dir` specifies where the command runs and `--path` is the path bundler will install things into. Probably missed that in the other PR.